mercury/models/src/main/scala/org/hyperledger/identus/mercury/model/error/package.scala

```diff
@@ -1,28 +1,11 @@
-package org.hyperledger.identus.mercury
-
-import java.io.IOException
+package org.hyperledger.identus.mercury.model
 
 package object error {
 
-  type MercuryException = MercuryError | IOException
-  type MercuryThrowable = MercuryError | IOException | Throwable // REMOVE Throwable
-
-  def mercuryErrorAsThrowable(error: MercuryThrowable): java.lang.Throwable = error match
-    // case ex: MercuryError =>
-    //   ex match
-    //     case te: TransportError => new RuntimeException(te)
-    case ex: IOException => ex
-    case ex: Throwable   => ex
-
   sealed trait MercuryError
```

Comment:
I will extends `org.hyperledger.identus.shared.models.Failure` on my PR.
Or are you also doing that on your next PR?